src/test/rgw/test_rgw_lua.cc

```diff
@@ -646,3 +646,20 @@ TEST(TestRGWLua, OpsLog)
   EXPECT_TRUE(olog.logged);
 }
 
+TEST(TestRGWLua, TracingAddEvent)
```

Comment:
please also add an ``AddAttribute()`` test
and a test covering the invalid cases where you raise an error.
you can verify that by checking the return value from ``execute()`` to be not zero